### PR TITLE
Add keepalive document

### PR DESCRIPTION
**Pull request by ganmacs**
ref: https://github.com/fluent/fluentd/pull/2393

### CODE REVIEW

File: docs/v1.0/out_forward.txt [hunk 1]
```diff
@@ -336,6 +336,22 @@ The client private key path for TLS.
 
 The client private key passphrase for TLS.
 
+### keepalive
+
+| type | default | version |
+|:----:|:-------:|:-------:|
+| bool | fales    | 0.14.3  |
```
Comment (cosmo0920) [inline docs/v1.0/out_forward.txt]: We should specify concrete version such as `1.4.3` in version column.

File: docs/v1.0/out_forward.txt [hunk 2]
```diff
@@ -336,6 +336,22 @@ The client private key path for TLS.
...
+
+Enable keepalive connection.
+
+### keepalive_timeout
+
+| type | default | version |
+|:----:|:-------:|:-------:|
+| time | nil     | 0.14.3  |
```
Comment (cosmo0920) [inline docs/v1.0/out_forward.txt]: ditto.

File: docs/v1.0/out_forward.txt (hunk 1 above)
Comment (ganmacs) [reply docs/v1.0/out_forward.txt]: Thank you for your comment. As current fluentd looks v1.4.2(https://github.com/fluent/fluentd/releases/tag/v1.4.2). I changed the version to 1.4.3.

**Comment by fujimotos** [comment]
Merged. Thank you.

**Comment by fujimotos** [comment]
Side note: I ported this change to the new GitBook site in fluent/fluentd-docs-gitbook/pull/23.

**Pull request merged**